MailLibrary/Drivers/ImapDriver.php

```diff
@@ -529,18 +521,14 @@ public function uploadRawMessage($contentOfMessage)
 
```

Comment:
in `uploadRawMessage`: 4th parameter of `imap_append` is string so `$options` must be string.

Reply:
fixed